server/tests/api.test.js

```diff
@@ -6,9 +6,29 @@ const request = supertest(app);
 describe('API Endpoints', () => {
   it('Should retrieve the game info for a game based on a valid query parameter ID', async (done) => {
     const res = await request.get('/game_carousel_info?id=1');
-
     expect(res.status).toBe(200);
-    expect(res.body.length).toBeGreaterThan(0);
+    expect(res.body.length).toEqual(1);
+    done();
+  });
+  it('Should retrieve the game info for a game based on a valid query parameter ID', async (done) => {
```

Comment:
need to change this to "should have a status of 500 for a query parameter ID that is invalid (not 1-100)" 😜